common/static/common/templates/discussion/new-post-visibility.underscore

```diff
@@ -0,0 +1,11 @@
+<% if (group_name) { %>
+    <%-
+    interpolate(
+        gettext('This post is visible only to %(group_name)s.'),
```

Comment:
This should be "This post **will be** visible only to..."

Same thing on line 10